### PR TITLE
Allow update packages in 4.3

### PR DESCRIPTION
**Pull request by jirimacku**
Since the last change we are missing the update of all packages before engine_setup.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by jirimacku** [comment]
ci test please
ci add to whitelist

File: defaults/main.yml [hunk 1]
```diff
@@ -100,6 +100,8 @@ he_tcp_t_port: null
 # ovirt-hosted-engine-setup variables
 he_just_collect_network_interfaces: false
 he_libvirt_authfile: '/etc/ovirt-hosted-engine/virsh_auth.conf'
+he_offline_deployment: false
+he_additional_package_list: []
```
[the remark this answers is not in the document]
Comment (jirimacku) [reply defaults/main.yml:104]: Done

File: tasks/full_execution.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@
         ovirt_engine_setup_answer_file_path: /root/ovirt-engine-answers
         ovirt_engine_setup_use_remote_answer_file: true
         ovirt_engine_setup_update_all_packages: false
-        ovirt_engine_setup_offline: true
+        ovirt_engine_setup_offline: "{{ he_offline_deployment }}"
         ovirt_engine_setup_admin_password: "{{ he_admin_password }}"
```
[the remark this answers is not in the document]
Comment (jirimacku) [reply tasks/full_execution.yml:41]: Done

**Comment by sandrobonazzola** [comment]
ci add to whitelist

**Pull request merged**